content/zh/docs/concepts/extend-kubernetes/api-extension/custom_resources.md

```diff
@@ -0,0 +1,524 @@
+-->
+|缩放子资源|允许水平Pod自动缩放器和Pod中断预算等系统与您的新资源进行交互|是的|是的|
+<!--
+| Status Subresource | <ul><li>Finer-grained access control: user writes spec section, controller writes status section.</li><li>Allows incrementing object Generation on custom resource data mutation (requires separate spec and status sections in the resource)</li></ul> | [Yes](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#status-subresource) | Yes |
+-->
+|状态子资源| ·细粒度访问控制：用户写入规范部分，控制器写入状态部分。 ·允许递增对象在自定义资源数据突变生成。（需要资源中的单独规范和状态部分）|是的|是的|
+<!--
+| Other Subresources | Add operations other than CRUD, such as "logs" or "exec". | No | Yes |
+-->
+|其他子资源|添加 CRUD 以外的操作,如"日志"或"执行"。|不是|是的|
+<!--
+| strategic-merge-patch | The new endpoints support PATCH with `Content-Type: application/strategic-merge-patch+json`. Useful for updating objects that may be modified both locally, and by the server. For more information, see ["Update API Objects in Place Using kubectl patch"](/docs/tasks/run-application/update-api-object-kubectl-patch/) | No | Yes |
+-->
+|战略-合并-补丁|新的端点支持具有Content-Type: application/strategic-merge-patch+json.的补丁。可用于更新可在本地和服务器修改的对象。更多信息，请参阅《使用kubectl修补程序更新API对象》|不是|是的|
+<!--
+| Protocol Buffers | The new resource supports clients that want to use Protocol Buffers | No | Yes |
+-->
+|协议缓冲区|新资源支持希望使用协议缓冲区的客户端|不是|是的|
+<!--
+| OpenAPI Schema | Is there an OpenAPI (swagger) schema for the types that can be dynamically fetched from the server? Is the user protected from misspelling field names by ensuring only allowed fields are set? Are types enforced (in other words, don't put an `int` in a `string` field?) | No, but planned | Yes |
+-->
+|开放API模版|是否有可从服务器动态提取类型的开放API模版？用户通过确保只设置允许的字段来避免拼写错误的字段名称是否被设置？是否是强制执行类型（换句话说，不要在字符串字段中放置int？）|不，但有计划|是的|
+<!--
+### Common Features
+-->
+常见功能
+<!--
+When you create a custom resource, either via a CRDs or an AA, you get many features for your API, compared to implementing it outside the Kubernetes platform:
+-->
+与在Kubernetes平台之外实现它相比，当通过CRD或AA创建自定义资源时，您可以获得API的许多功能：
+<!--
+| Feature | What it does |
+-->
+|功能|作用|
+| ------- | ------------ |
+<!--
+| CRUD | The new endpoints support CRUD basic operations via HTTP and `kubectl` |
+-->
+|CRUD|通过HTTP和kubectl，新的端点通过HTTP和kubectl支持CRUD基本操作|
+<!--
+| Watch | The new endpoints support Kubernetes Watch operations via HTTP |
+-->
+|监视|新的端点通过HTTP支持Kubernetes监视功能|
+<!--
+| Discovery | Clients like kubectl and dashboard automatically offer list, display, and field edit operations on your resources |
+-->
+|发现|如Kubectl和dashboard客户端会自动提供资源上的列表、显示和字段编辑操作|
+<!--
+| json-patch | The new endpoints support PATCH with `Content-Type: application/json-patch+json` |
+-->
+| json-patch|新的端点支持具有内容类型的补丁：应用程序/ json-patch+json|
+<!--
+| merge-patch | The new endpoints support PATCH with `Content-Type: application/merge-patch+json` |
+-->
+| merge-patch|新的端点支持具有内容类型的补丁：应用程序/ merge -patch+json|
```

Comment:
I'd expect this line to include the HTTP verb `PATCH`, and the correct media type.